spec/lib/tasks/import_spec.rb

```diff
@@ -53,9 +53,13 @@
     end
 
     it 'runs the number_of_views_by_organisation task' do
+      @cached_pageview_batch_size = ENV["GA_PAGEVIEW_BATCH_SIZE"]
```

Comment:
@chao-xian No it doesn't. You caught my copy and paste error. Have fixed and fixed up.